docs/reference/triggers/v3iostream.md

```diff
@@ -100,56 +100,84 @@ As of Nuclio v1.1.33 / v1.3.20, you can configure the following configuration pa
+After the command completes successfully, the stream is ready for consumption by a Nuclio function. To test this, do the following from the dashboard's **Projects** page, to define and deploy a function that consumes the stream records and uses a `v3ioStream` trigger.
 
-```python
-import time
+> **Note:** Before you deploy the function, ensure that the log-forwarder service is enabled (see the **Services** dashboard page), so that you can view the function logs.
 
+1.  Select and existing project or create a new project, and then create a new Python function. Select to create the function from scratch.
 
-def handler(context, event):
-    context.logger.debug_with('Got event', shard_id=event.shard_id, body=event.body.decode('utf-8'))
-    time.sleep(5)
-```
+2.  On the function page, in the **Code** tab, set **Code entry type** to "Source code (edit online)" (default), and enter the following code in the **Source code** text box to define a function that logs the shard-ID event body and sleeps for 5 seconds:
+    ```python
+    import time
+
+
+    def handler(context, event):
+        context.logger.debug_with('Got event', shard_id=event.shard_id, body=event.body.decode('utf-8'))
+        time.sleep(5)
+    ```
+
+3.  Select the **Triggers** tab, and then select **Create a new trigger**.
+    In the **Name** text box, enter the trigger name "V3IO stream", and in the **Class** field select "V3IO stream" from the drop-down list.
+    Use the following trigger configuration:
 
-Under `Triggers`, create a `V3IO stream` with the following configuration:
-* URL: http://v3io-webapi:8081/users/test-stream-0@cg0
-* Max Workers: 8 (can be more, but here we'll have 8 workers reading all the shards)
-* Seek To: Earliest (if we use `Latest`, we'll never read the records we put into the stream - only records we create after deploy)
-* Password: Your access key
+    - **URL** - `http://v3io-webapi:8081/users/test-stream-0@cg0`.
+    - **Max Workers** - `8`.
+      (You can also set this to a higher number, but then you'll have 8 workers reading all the shards.)
+    - **Seek To** - `Earliest`.
+      (If you use `Latest`, only records that are ingested after the function is deployed will be read, so you won't read the records that you already added in the previous step.)
+    - **Password** - a valid platform access key.
 
-Leave everything else default / empty and deploy (if Partitions requires a value, just put `0` there). Follow the logs of the function pods and watch the events being handled. The logs should contain:
+    For all other configuration fields, use the default configuration.
+    (If you're required to set **Partitions**, enter `0`).
+
+4.  Select **Deploy** to deploy your function.
+
+After the deployment succeeds, check the logs of the function pods (see the **Logs** dashboard page) and watch the events being handled (see the **Events** dashboard page). The logs should contain the following:
```

Comment:
Again, logs thingie

Reply:
I edited the paragraph — see commit f61916f.